### ISSUE TITLE
Redundant call to CloudFront getCachePolicy

### ISSUE THREAD

**Issue by Georgeforman3** [iann0036/former2#239]
Hi,
In cloudfront.js file there's a call to 'listCachePolicies' and then for each policy another call to 'getCachePolicy', but it seems that the listCachePolicies returns the same data as get.
This affects rate limits on my AWS account, any chance to remove the redundant call or am I missing anything?

**Comment by iann0036** [comment]
Hi @Georgeforman3,

Sorry for the rate limiting issues. The list-then-get pattern is in place in as much places as possible within Former2 to maintain consistency with other calls. It's somewhat rare that the list call contains all required data and this also may change with updates.

**Comment by Georgeforman3** [comment]
Thanks @iann0036! another question about the list-then-get pattern, I see that for all resource type you store the 'get' object on the 'f2data' field and later on use it to generate the cfn/tf resource, but for CloudFront Distribution you use the 'list' response object and you store it and not the 'get' object. Is that on purpose?
It seems like that's the only place you do that.

**Comment by Georgeforman3** [comment]
I've looked more into the CloudFront API and the Logging fields are missing from the 'ListDistributions' response so the CFN resource being generated might cause a drift:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-logging.html

**Comment by iann0036** [comment]
Hey @Georgeforman3,

Thanks for the heads up! I'll adjust that call to be Inline with the others in a bit.

**Comment by Georgeforman3** [comment]
Hi @iann0036  - have you had the chance to adjust the call? Thanks again!

**Comment by iann0036** [comment]
Hey @Georgeforman3,

Sorry for the delay. I've adjusted the call to use the get response object as you pointed out. CloudFront was one of the [first](https://github.com/iann0036/former2/commit/258929e2f9673a6055ccf2c133197d167733529b#diff-b4075f409bf946eda16c69a6f73c92ad0dafaed7aa81368863e355172f668fa0R3716) and more complicated resources to deal with, so there was some legacy.

I've added the `Logging` property also for you. Let me know how that all goes.
